apps/docs/content/nodejs/how-to/env-variables.mdx

```diff
@@ -195,3 +207,24 @@ You can use runtime env variables in the build environment using the `RUNTIME_`
 If you create a secret env variable and a basic runtime env variable with the same key, Zerops saves the basic runtime env variable from your zerops.yml and ignores the secret env variable.
 
 If you create a basic build env variable and a runtime env variable with the same key, Zerops saves both because the build and runtime environments have separate sets of env variables.
+
+## Setting secret variables
```

Comment:
Take only line 217 - 226 and add them to line 83 with a title ### Set secret env variables when importing a service